src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from "react";
+import "./footer.module.scss";
+
+const Footer: React.FC = () => {
```

Comment:
In this case we can remove React.FC, because there are no props here.

Reply:
Fixed